src/compiler/lsp/lsp.h

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2019 Toitware ApS.
```

Comment:
Mostly fresh code.

It introduces an `Lsp` class as facade for all Lsp operations.